release-02/code/flag_tool.rb

```diff
@@ -0,0 +1,38 @@
+require 'sequel'
+
+helpText = "ITU-Minitwit Tweet Flagging Tool\n\n" +
```

Comment:
In Ruby it is convention to use `snake_case` for variables and methods/functions. Don't worry about this for now. We will revisit this, once we have `Rubocop` in the project. When integrated into VS Code it will tell you what parts of the code are not aligned with the conventions. VS Code can then also format it automatically.
[Ruby VS Code extension](https://marketplace.visualstudio.com/items?itemName=rebornix.Ruby). I can assist in setting this up for everybody.